### DISCUSSION TITLE
New F Prime Website is Live

### DISCUSSION THREAD

**Discussion by thomas-bc** [nasa/fprime#3064] [Announcements]
# Announcement

Another exciting news: F Prime just released its new website! 🚀 

Please head over to https://fprime.jpl.nasa.gov to find our documentation, informational content and more, all in one place.

Upgrades include the addition of a search bar and a new information architecture for the documentation. And, of course, the obligatory dark theme.

If you have any feedback or suggestions, please let us know!

## More details

New features include:

- New Information Architecture – based on the [Diataxis framework](https://diataxis.fr/)
- Robust search capabilities
- Easy navigation with a sidebar menu, sub-pages, and a table of contents on each page
- New informational pages about the framework
- Improved Code Blocks – code blocks are now stylized with proper syntax highlighting, code blocks can also easily be copied with a click of a button
- Light/Dark mode

**Comment by kitty99099** [comment]
Looking forward the renewal of OSAL!

**Comment by keck-in-space** [comment]
This rules! Thank you for porting all this over!
